src/Command/CheckLocalizationCommand.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace PHPCensor\Command;
+
+use Monolog\Logger;
+use PHPCensor\Service\BuildService;
+use PHPCensor\Store\Factory;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Check localizations.
+ */
+class CheckLocalizationCommand extends Command
+{
+    /**
+     * @var array
+     */
+    protected $excluded = ['lang.en.php'];
+
+    /**
+     * Construct.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->basePath = __DIR__.'/../Languages';
+    }
+
+    /**
+     * Configure.
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('php-censor:check-localizations')
+            ->addOption('same', 0, InputOption::VALUE_OPTIONAL, 'Same than English version (0 = no, 1 = yes)')
+            ->addOption(
+                'langs',
+                [],
+                InputOption::VALUE_OPTIONAL,
+                'List of languages separated by commas. By default, all languages'
+            )
+            ->setDescription('Check localizations.');
+    }
+
+    /**
+     * Loops through running.
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $output->writeln('');
+        $output->writeln('<info>Check localizations!</info>');
+        $output->writeln('');
+
+        $sameThanEnglish = (isset($input->getOption('same'))) ? $input->getOption('same') : false;
```

Comment:
PHP Fatal error:  Cannot use isset() on the result of a function call (you can use "null !== func()" instead) in /home/travis/build/php-censor/php-censor/src/Command/CheckLocalizationCommand.php on line 63